measurement/validation.py

```diff
@@ -33,18 +36,25 @@ def get_data_to_validate(
     start_time_ = datetime.strptime(start_time, "%Y-%m-%d").replace(tzinfo=tz)
     end_time_ = datetime.strptime(end_time, "%Y-%m-%d").replace(tzinfo=tz)
     extra = {}
-    if not include_validated:
+    if only_validated:
+        extra = {"is_validated": True}
+    elif not include_validated:
         extra = {"is_validated": False}
 
-    return pd.DataFrame.from_records(
+    df = pd.DataFrame.from_records(
         Measurement.objects.filter(
             station__station_code=station,
             variable__variable_code=variable,
             time__gte=start_time_,
             time__lte=end_time_,
             **extra,
         ).values()
-    ).sort_values("time")
+    )
+
+    if df.empty:
+        return pd.DataFrame()
```

Comment:
Just to confirm I get this right. If `df.empty`, that means it has the right column headers but there are no records. `pd.DataFrame()` is totally empty - no columns or records in it. Is this correct? If that's the case, why a totally empty dataframe is preferable?

Reply:
Good point. `df.empty` just checks that there are no records, but in this case an empty dataframe will also have no columns. There is no difference with `pd.DataFrame()` so I can just return `df`. Not sure why I wasn't doing this to begin with